src/backend/distributed/connection/placement_connection.c

```diff
@@ -1175,6 +1115,19 @@ InitPlacementConnectionManagement(void)
 }
 
 
+/*
+ * UseConnectionPerPlacement returns whether we should use as separate connection
+ * per placement even if another connection is idle. We mostly use this in testing
+ * scenarios.
+ */
+bool
+UseConnectionPerPlacement(void)
```

Comment:
maybe `CanUseConnectionPerPlacement` or `ShouldUseConnectionPerPlacement` as it does not change the state

Reply:
We decided to rename `CONNECTION_PER_PLACEMENT` to `REQUIRE_CLEAN_CONNECTION`
Thanks for reminding me that there are other places where I need to rename.

How does eda7d8e0d2b6281b68223f3b78d4f9c79a53be0d look? I renamed this function to `ShouldUseCleanConnection`

Reply:
looks good as the enum implies